.github/workflows/php.yml

```diff
@@ -0,0 +1,74 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: ''
+            composer-flag: ''
```

Comment:
Just checking: is it really necessary to have these empty vars (`stability` and `composer-flag`) or can you leave them out?

Reply:
No, it seems to be unnecessary :)